### ISSUE TITLE
Modal is not fully accessible according to WCAG

### ISSUE THREAD

**Issue by sbrigham** [jossmac/react-images#270]
https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html#keyboard-operation-trapping-examples-head

When the modal is open, the focus should be trapped inside of the modal, until the modal is closed. The next, previous, and escape keys should be tabbable as well.

**Comment by neptunian** [comment]
Please see the new version
